examples/neo-place/secrets.yaml

```diff
@@ -0,0 +1,4 @@
+env_variables:
```

Comment:
question: Is there any security downside if we include this file?

Reply:
yes there are, and my bad for committing this, I'll change the keys and remove this file